.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!-- 
+Insert your contribution description here.
+Ensure your contribution fills the Contributing Guidelines described here:
+https://github.com/bonitasoft/bonita-doc/blob/7.3/CONTRIBUTING.md 
+-->
+
+<!-- 
+Special Attention to TITLE format
+https://github.com/bonitasoft/bonita-doc/blob/7.3/CONTRIBUTING.md#title
```

Comment:
`title format is more for commits than for PR`.... I do agree however, PR is the best way we have to give a hint about best practices.